qpc/release.py

```diff
@@ -1,9 +1,11 @@
 """File to hold release constants."""
+import os
+
 from . import __package__version__
 
 VERSION = __package__version__
 AUTHOR = "QPC Team"
 AUTHOR_EMAIL = "qpc@redhat.com"
-PKG_NAME = "qpc"
-ENTRYPOINT = f"{PKG_NAME}=qpc.__main__:main"
+QPC_VAR_PKG_SOURCE = os.environ.get("QPC_VAR_PKG_SOURCE", "qpc")
```

Comment:
why not `QPC_VAR_PKG_NAME`

Reply:
or, even better, `QPC_VAR_PROGRAM_NAME` as it is declared on the jinja template?

Reply:
agreed!